Dynatrace/dynatrace-operator
# Add gke-autopilot namespaces to default ignored namespaces

## Description

This is a change to make gke-autopilot run, because previously when the operator was deployed things would not work as there would be an error message that certain namespaces (with prefix gke- and gmp-) cannot be updated or changed in any way. That is why those are added now to the ignored namespaces.

Also, in order to make the e2e tests run I added a timeout in wait.go because when I ran the e2e tests on gke-autopilot it took a lot of time and it timed out almost everytime. When this was fixed I had problems with the sample pods which where randomly stuck in a ContainerStatusUnknown state. This is fixed by adding ephemeral storage of 2Gi to the sample app manifest.

## How can this be tested?

Create a gke-autopilot cluster and run the tests that should be possible (application monitoring default for example)

## Checklist
